frontend/src/pages/Landing/index.js

```diff
@@ -40,16 +43,14 @@ export function renderTotal(offset, perPage, activePage, reportsCount) {
   return `${from}-${to} of ${reportsCount}`;
 }
 
-function regionFilter(regionId) {
-  return {
-    topic: 'region',
-    condition: 'Contains',
-    query: regionId.toString(),
-  };
-}
-
 function Landing({ user }) {
+  // Determine Default Region.
   const regions = allRegionsUserHasPermissionTo(user);
+  const regionToUse = regions[0] || user.homeRegionId || 0;
```

Comment:
I think we want to flip `regions[0]` with `user.homeRegionId` so we use `homeRegionId` if we can, the first region numerically if `homeRegionId` isn't set, and fall back to 0 if they have no permissions on any region. Honestly we can probably remove the `|| 0` completely and treat users with no home region and no premissions to read any region as a "central office" user that just sees nothing on this page.

If we keep this as is a user with a `homeRegionId` of 2 and permission to read both region 1 and 2 will default to region 1 on this page.

Reply:
So do you think the code should just become this?
`
 const defaultRegion = user.homeRegionId || regions[0];
`

Reply:
I _think_ so, assuming the page doesn't just explode for a user with zero read permissions